tsconfig.json

```diff
@@ -4,7 +4,7 @@
 		"experimentalDecorators": true,
 		"forceConsistentCasingInFileNames": true,
 		"isolatedModules": true,
-		"lib": ["es2022", "esnext.disposable", "DOM"],
+		"lib": ["es2022", "DOM"],
```

Comment:
Confirmed this is not a valid value for this field: https://www.typescriptlang.org/tsconfig/#lib
Lint error:
<img width="1432" height="448" alt="image" src="https://github.com/user-attachments/assets/5cac8c53-4b2d-4585-8e78-8b6ab3f04720" />